src/main/java/com/mraof/minestuck/world/storage/ClientPlayerData.java

```diff
@@ -29,7 +29,7 @@
 	
 	private static Modus modus;
 	private static Title title;
-	private static int rung;
+	public static int rung;
```

Comment:
was this change intentional?

Reply:
idk

Reply:
It shouldn't have been, since you can do ClientPlayerData.getRung() instead of accessing it directly.